build/common.sh

```diff
@@ -36,7 +36,12 @@ while getopts ${OPTS} OPT; do
 		export PORTSREFURL=${OPTARG}
 		;;
 	a)
-		export PRODUCT_TARGET=${OPTARG%%:*}
+		TARGET=${OPTARG%%:*}
+		if [ ${TARGET} == aarch64 ]; then
```

Comment:
what is being passed from the command line? Ideally we only call `make arm DEVICE=RPI3` and that would take care of all of this without the need of immediate input rewrite

Reply:
Well, the `ARCH` got by `Makefile` is through command `uname -p`, and passed to `common.sh` as `PRODUCT_TARGET `.
On aarch64 machines, the result of `uname -p` is `aarch64`, while we need `arm64` as `PRODUCT_TARGET ` to build aarch64 targets.